app.py

```diff
@@ -0,0 +1,45 @@
+from flask import Flask, request, jsonify
+from flask_migrate import Migrate
+from models.model import db, User
+
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///database.db'
+app.config['SECRET_KEY'] = 'gunjan'
+db.init_app(app) 
+migrate = Migrate(app, db)
+
+
+
+@app.route('/', methods=['POST'])
+def register():
```

Comment:
Add a condition to check user is already registered or not, if not then register, else return a message "User is already registered with this email".